.netlify/state.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I would add this file to your gitignore. You'll probably need to remove the cached file from github.

Reply:
Okay will do. Thanks